packages/ui/tailwind.config.js

```diff
@@ -38,6 +38,19 @@ const tertiary = {
 
 module.exports = {
   theme: {
+    opacity: {
+      '10': '0.1',
+      '20': '0.2',
+      '30': '0.3',
+      '40': '0.4',
+      '50': '0.5',
+      '60': '0.6',
+      '70': '0.7',
+      '80': '0.8',
+      '90': '0.9',
+      '100': '1',
```

Comment:
Were these not already in here by default? Why are we adding so many

Reply:
I have removed these since they weren't being used